api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/CohortListEntryTranslatorImpl.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import java.util.List;
+
+import org.hl7.fhir.r4.model.ListResource;
+import org.openmrs.Cohort;
+import org.openmrs.module.fhir2.api.translators.ListEntryTranslator;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CohortListEntryTranslatorImpl implements ListEntryTranslator<Cohort> {
+	
+	@Override
+	public List<ListResource.ListEntryComponent> toFhirResource(Cohort cohort) {
+		return null;
+	}
```

Comment:
You shouldn't be returning null. Instead
`throw new NotImplementedException();`

Reply:
The suppose of this class is only to create a base from which we can provided implementation in the other supported versions

Reply:
Then just use an interface instead